src/frontend/styles/components/navigator.pcss

```diff
@@ -0,0 +1,40 @@
+.navigator {
```

Comment:
margin above the navigation is missed 
<img width="989" alt="image" src="https://user-images.githubusercontent.com/3684889/181501836-7b56fef3-91de-42e3-a1ec-4904a685fe58.png">